docs/01-Welcome-to-FOME.md

```diff
@@ -119,8 +119,6 @@ The first page to view should probably be [the FOME Overview page](01-Intro-Star
 - [Idle VE Table](Basic-Features/Idle-Control/Idle-VE-Table)
 - [Idle tuning](Basic-Features/Idle-Control/Idle-tuning)
 
----
```

Comment:
this confuses me; these create horizontal rules in the page.  we can't use these anymore?

Reply:
the horizontal rulers are ok, I was trying to find the root cause for a link error, will re-establish them after root cause is identified and fiyed